packages/components/src/components/button/button.custom.styles.ts

```diff
@@ -1,5 +1,133 @@
 import { css } from 'lit';
 
 export default css`
-  /* Write custom CSS here */
-`;
+.button:focus-visible {
+    outline: var(--syn-focus-ring-color) solid 2px;
+    outline-offset: 2px;
+  }
+
+  /*
+ * Size modifiers
+ */
+
+  .button.button--medium.button--has-label .button__label {
+    font-size: var(--syn-font-size-medium);
+   } 
+
+  .button.button--large.button--has-label .button__label {
+    font-size: var(--syn-font-size-large);
+  }
+
+  /*
+ * Standard buttons
+ */
+
+  .button--filled.button--primary.button--disabled {
```

Comment:
Why did the other button variants (e.g. accent, secondary, danger, ...) disappear? Was this intentional? Our current Design system ships applications that make use of those, so it would be pretty breaking change to just have text/outline/default available instead of different colors.

Reply:
We plan to provide more colors in future, but currently don't provide them in design as well. We have to clarify stuff like a11y of those colors, if provide error and danger colors as well.

I personally would provide every color as well for filled, outline and text.

Reply:
Yes. You have a danger button on your side? That's interesting, because it's not on brand portal AFAIK.

Reply:
I think that this is a great idea, giving much more freedom for forming out what you want to tell the developer, instead of giving him some 12 or 14 different button shapes. That said, we did not even multiply those out in our current system, so people added feature requests for primary-outline-without-text-but-only-on-hover (you name it) ;)

Reply:
Mario explainded it very good, thanks.

We also wanted to remove the orange button for a11y reasons.

Red and green buttons are not a requiremet I've heared, but easy to add if the applications needs it somewhere eg. some dangerous deletion thing.